libraries/aws_iam_policies.rb

```diff
@@ -26,7 +26,14 @@ class AwsIamPolicies < AwsResourceBase
 
   def initialize(opts = {})
     super(opts)
-    validate_parameters
+    validate_parameters(allow: %i(only_attached scope))
+    @only_attached = opts[:only_attached]
```

Comment:
Does only_attached need to be set here as a global var? Would the behaviour be the same if line 34 was     @parameters[:only_attached] = opts[:only_attached] if opts[:only_attached]